### DISCUSSION TITLE
Ready for production?

### DISCUSSION THREAD

**Discussion by NikolayZobnev** [WJSoftware/vite-plugin-single-spa#109] [General]
Hello!

I am planning to use your plugin in production.

Can you please tell me how ready you think your solution is for production?

Maybe I need to pay attention to something and help in finalizing it?

p.s at the moment made small demos with your plugin.

**Comment by webJose** [comment]
Hello and welcome. I've been using the plug-in at work in production since January and works just fine. With the bug fix in v0.6.1 the plug-in is quite decent, I would say. 

If you're looking for the date when it goes out of experimental phase, I don't know it. It will continue to have this label until I'm fully satisfied with its API.

If you have a particular concern, I'd suggest that you test the scenario yourself and bring any issues you find here in the form of a new issue. If your scenario is satisfactory then great.

Finally, any suggestions for improvement and new features are welcome.
